Cargo.toml

```diff
@@ -228,6 +228,11 @@ strip = 'none'
 # For convenience.
 strip = 'none'
 
+# The Class-Groups key generation output was too large, causing runtime stack overflow errors. This optimization resolved the issue
+# Note: This flag causes issues with the debugger (#441)
+#[profile.dev]
```

Comment:
I ran the project without this flag and with the `mock-class-groups` feature, and it works. I will run it again without the feature to check if it is no longer needed, but this process takes some time, and I don’t see a reason to delay this PR.

Reply:
I checked. This flag is no longer needed, but without it on "dev" mode generating key-pair and proof takes hours. 
Should I keep it for now? @zeevmoney